### PR TITLE
Resolve AFP toggle naming confusion in crash index advanced filters

### PR DESCRIPTION
**Pull request by roseeichelmann**
## Associated issues
Closes https://github.com/cityofaustin/atd-data-tech/issues/11819

## Testing
**URL to test:** 
[Netlify](https://deploy-preview-1200--atd-vze-staging.netlify.app/#/)

**Steps to test:**
1. Go to the crashes page advance filters section, see the new toggle label that says "Include Outside Of Austin Full Purpose".

---
#### Ship list
- [x] Code reviewed
- [x] Product manager approved

### CODE REVIEW

**Review by patrickm02L — APPROVED** [review]
Thanks for knocking this out! 🥊 

Screen Shot 2023-03-24 at 2 17 01 PM

File: atd-vze/src/views/Crashes/crashGridTableParameters.js [hunk 1]
```diff
@@ -244,7 +244,7 @@ export const crashGridTableAdvancedFilters = {
       },
       {
         id: "geo_afd",
-        label: "Remove Austin Full Purpose",
+        label: "Include Outside Of Austin Full Purpose",
```
Comment (mddilley) [inline atd-vze/src/views/Crashes/crashGridTableParameters.js:247]: I don't think that the VZ team will have any confusion here now! Super small detail, but I noticed that there is inconsistent capitalization on one toggle label - `Has been Geocoded`. Not a big deal but wondering if we should patch that one up while you are in here.

**Review by mddilley — APPROVED** [review]
Yay for clearer labels! 🚀 🚢 🙏

File: atd-vze/src/views/Crashes/crashGridTableParameters.js (hunk 1 above)
Comment (frankhereford) [reply atd-vze/src/views/Crashes/crashGridTableParameters.js:247]: I think this is a great idea. Polish to the UI is never a bad thing!

**Review by frankhereford — APPROVED** [review]
Big 👍 from me. Copy edit in the code looks good, and it continues to function as expected over different break points as I resize the window. This is a 🚀.

File: atd-vze/src/views/Crashes/crashGridTableParameters.js (hunk 1 above)
Comment (roseeichelmann) [reply atd-vze/src/views/Crashes/crashGridTableParameters.js:247]: thanks for catching that mike, just fixed that capitalization inconsistency!

**Review by chiaberry — APPROVED** [review]

**Pull request merged**